PolicyProject1/src/main/java/com/example/demo/model/Policy.java

```diff
@@ -1,84 +1,69 @@
 package com.example.demo.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import java.time.LocalDateTime;
 
+/**
+ * this class uses lombok annotations.
+ * Refer: https://www.baeldung.com/lombok-ide for more info
+ */
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
 @Entity
 public class Policy
-{	
+{
 	@Id
 	@GeneratedValue(strategy=GenerationType.AUTO)
 	private int id;
-	private int time_period;
-	private String inception_date;
-	private String expiry_date;
-	
-	//monthly,quartly,yearly
-	private String maturity_type;
-	private String policy_state;
-	
-	//one policy have one product
-//	private Product product;
-	
-	//getter and setter of id
-	public void setId(int id)
-	{
-		this.id=id;
-	}
-	public int getId()
-	{
-		return id;
-	}
-	
-	//getter and setter of time_period
-	public void setTime_period(int time_period)
-	{
-		this.time_period= time_period;
-	}
-	
-	//getter and setter of inception_date
-	public void setEncepion_date(String inception_date)
-	{
-		this.inception_date = inception_date;
-	}
-	
-	public String getInception_date()
-	{
-		return inception_date;
-	}
-	
-	//getter and setter of expiry_date
-	public void setExpiryDate(String expiry_date)
-	{
-		this.expiry_date= expiry_date;
-	}
-	
-	public String getExpiryDate()
-	{
-		return expiry_date;
-	}
-	
-	//getter and setter of maturity type
-	public void setMaturity_Type()
-	{
-		this.maturity_type = maturity_type;
-	}
-	
-	public String getMaturity_type()
-	{
-		return maturity_type;
-	}
-	@Override
-	public String toString() {
-		return "Policy [id=" + id + ", time_period=" + time_period + ", inception_date=" + inception_date
-				+ ", expiry_date=" + expiry_date + ", maturity_type=" + maturity_type + ", policy_state=" + policy_state
-				+ "]";
-	}
-	
-	
-	
-	
 
+	/**
+	 * Unit in days
+	 */
+	private long policyLife;
+
+	/**
+	 * Total amount insured
+	 */
+	private Double sumInsured;
+
+	/**
+	 * Premium to be paid in each premiumTerm
+	 */
+	private Double premium;
+
+	/**
+	 * How frequently premium needs to be paid
+	 */
+	private PaymentTerm premiumTerm;
+
+	/**
+	 * Creation date of policy
+	 */
+	private LocalDateTime createdAt;
+	/**
+	 * Last update date of policy
+	 */
+	private LocalDateTime updatedAt;
+
+	/**
+	 * Inception date of policy
+	 */
+	private LocalDateTime inceptionDate;
+
+	/**
+	 * Expiry date of policy
+	 */
+	private LocalDateTime expiryDate;
+
+	/**
+	 * Current policy state
+	 */
+	private PolicyState state;

```

Comment:
here we are only defining variable names not getter setter for this.

Reply:
Also, see how I've added comments for them and the indent behind renaming the variables